velox/experimental/wave/common/tests/CMakeLists.txt

```diff
@@ -26,6 +27,7 @@ add_executable(
 add_test(velox_wave_common_test velox_wave_common_test)
 set_tests_properties(velox_wave_common_test PROPERTIES LABELS cuda_driver)
 
+target_include_directories(velox_wave_common_test PRIVATE ../../../breeze)
```

Comment:
Better to use a header-only library for CMake.
https://discourse.cmake.org/t/header-only-library/5476

Reply:
I will leave this to @dreveman to encapsulate the library in a cmake project

Reply:
Yep, sounds like a good thing to do as a follow up.